### PR TITLE
Add the ability to claim and load chunks with 1 click

### LINKED ISSUE #14411: FTBU skip claiming to load
### Your GTNH Discord Username

Santa's Child#0408

### Your Pack Version

2.4.0 RC2

### Your Proposal

allow holding ctrl to let you skip claiming the chunk and just claim and load all at once

### Your Goal

streamline use of FTBU

### Your Vision

would make using ftbu a nicer smoother experience

(added here due to no issues on FTBU)

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying *I like this feature please reconsider adding it* will prompt us to investigate and reopen it once we confirm your report.
- [X] I believe there is nothing similar in the pack already, or the existing solution isn't good enough.
- [X] I understand this change request may not attract enough attention and thus not be implemented.
- [X] I understand this change request may be rejected due to other community members thinking it's inappropriate.
- [X] I believe this feature would make the pack better.

### PR DESCRIPTION
**Pull request by Lyfts**
Adds the ability to go directly from unclaimed to claimed & chunkloaded by clicking CTRL + LClick  on an unclaimed chunk.
Replaces the old load function since it serves the exact same function on an already claimed chunk.

I have tested it a fair bit and haven't ran into any issues so far.

Closes: https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/14411

### CODE REVIEW

File: src/main/java/ftb/utils/net/MessageClaimChunk.java [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+import latmod.lib.ByteCount;
 
 public class MessageClaimChunk extends MessageFTBU {
 
     public static final int ID_CLAIM = 0;
     public static final int ID_UNCLAIM = 1;
     public static final int ID_UNCLAIM_ALL = 2;
     public static final int ID_UNCLAIM_ALL_DIMS = 3;
-    public static final int ID_LOAD = 4;
+    public static final int ID_CLAIM_AND_LOAD = 4;
```
Comment (BlueWeabo) [inline src/main/java/ftb/utils/net/MessageClaimChunk.java:16]: any reason you didn't go for a new ID? have you tries trying to load a chunk already claimed? could one accidentally claim a chunk of another person this way or load it up for them?

**Review by BlueWeabo — COMMENTED** [review]
Overall seems fine, but would it be possible to remove wildcard imports in the files you touched? (So GuiClaimChunks and MessageClaimChunk) Won't require you to go through all files spotless touched as that would be unfair.

File: src/main/java/ftb/utils/net/MessageClaimChunk.java (hunk 1 above)
Comment (Lyfts) [reply src/main/java/ftb/utils/net/MessageClaimChunk.java:16]: If you try to load an already loaded chunk nothing happens, that logic is being handled elsewhere. Same if you were to try to claim an already claimed chunk in the current version.

I can't imagine that you would be able to claim another persons chunk as that should be prevented by the same logic that prevents you claiming another person claim. But I will test that real quick to be 100% sure.

Comment (Lyfts) [reply src/main/java/ftb/utils/net/MessageClaimChunk.java:18]: Alright tested, you can't claim or load another persons chunks.

Comment (BlueWeabo) [reply src/main/java/ftb/utils/net/MessageClaimChunk.java:18]: Good to know. thanks

**Review by BlueWeabo — APPROVED** [review]
Thanks for your work

**Pull request merged**